test/extended/networking/multicast.go

```diff
@@ -167,5 +168,16 @@ func launchTestMulticastPod(f *e2e.Framework, nodeName string, podName string) (
 		podIP = pod.Status.PodIP
 		return (podIP != "" && pod.Status.Phase != kapiv1.PodPending), nil
 	})
+
+	if err != nil {
+		// Append error logs
+		logs, logErr := e2e.GetPodLogs(f.ClientSet, f.Namespace.Name, podName, fmt.Sprintf("%s-pod", podName))
+		if logErr != nil {
+			e2e.Failf("Error getting container logs: %s", logErr)
+		}
+
+		err = errors.New(err.Error() + "\nContainer Logs:\n" + logs)
```

Comment:
This expects caller to log the error, why not log the podLogs here itself just like what we do in other places?
Prefer e2e.Failf() or e2e.LogF(), check https://github.com/openshift/origin/pull/19130/files for reference.

Reply:
This will log more details for test case:
```
[Area:Networking] multicast when using one of the plugins 'redhat/openshift-ovs-multitenant, redhat/openshift-ovs-networkpolicy' should allow multicast traffic in namespaces where it is enabled [Suite:openshift/conformance/parallel] 5m10s
```
Don't we need similar code for this below test case?
```
NetworkPolicy when using a plugin that implements NetworkPolicy should enforce policy based on PodSelector [Feature:OSNetworkPolicy] [Suite:openshift/conformance/parallel] 1m12s
```

Reply:
> Don't we need similar code for this below test case?

as I said above, it's already there (it just postdates the rhbz bug)

Reply:
(It's also possible that the NetworkPolicy flake was fixed by #19080)